algorithms/src/fft/domain.rs

```diff
@@ -44,6 +44,9 @@ use std::fmt;
 #[cfg(feature = "parallel")]
 use rayon::prelude::*;
 
+#[cfg(not(feature = "parallel"))]
```

Comment:
Whats the purpose of the parallel cfg guard?

Reply:
(I presume it has something to do with Rayon compatibility)

Reply:
`zip_eq` is available via both `Itertools` and `rayon::ParallelIterator`. In `parallel` mode, we use the latter impl, while in `not(parallel)` mode, the former. To avoid an unused import warning when `parallel` is set, I guard `Itertools` with `not(parallel)`.